src/prompt.ts

```diff
@@ -38,9 +38,12 @@ export const createPromptWindow = async () => {
   promptWindow = new BrowserWindow({
     frame: false,
     transparent: true,
-    backgroundColor: '#00000000',
+    backgroundColor: nativeTheme.shouldUseDarkColors
+      ? '#33000000'
+      : '#C0FFFFFF',
+    vibrancy: nativeTheme.shouldUseDarkColors ? 'dark' : 'medium-light',
```

Comment:
This adds a nice vibrancy (blurred background) effect on MacOS. 
I'm using `shouldUseDarkColors` bool to switch between dark and light mode of the vibrancy effect.

I noticed that if I were to change my system's color preference, the app had to be restarted in order for it to take effect. I'm not sure whether that can be avoided. 🤔